app/gradescope_to_spreadsheet.py

```diff
@@ -0,0 +1,334 @@
+# Author: Naveen Nathan
+
+"""
+This file is deprecated. Please use 'api/app.py' instead.
+"""
+
+import json
+
+from fullGSapi.api import client
+import os.path
+import sys
+import re
+import pandas as pd
+import io
+import time
+import warnings
+import functools
+
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+from google_auth_oauthlib.flow import InstalledAppFlow
+from googleapiclient.discovery import build
+from googleapiclient.errors import HttpError
+
+# CS10 Fa24 course id
+COURSE_ID = "831412"
+# This scope allows for write access.
+SCOPES = ["https://www.googleapis.com/auth/spreadsheets"]
+SPREADSHEET_ID = "1Ms0PvwUKbph3p29bih-rDBV9EjyI9Ae-T3rGlRDwXqc"
```

Comment:
When you get the testing working, provide the spreadsheet with the output with student data and PII hidden.